From: Shengjing Zhu <zhsj@debian.org>
Date: Thu, 24 Feb 2022 02:03:08 +0800
Subject: Skip system bus test

Forwarded: not-needed

---
 conn_test.go | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/conn_test.go b/conn_test.go
index b73fddd..b61e258 100644
--- a/conn_test.go
+++ b/conn_test.go
@@ -33,6 +33,7 @@ func TestSessionBus(t *testing.T) {
 }
 
 func TestSystemBus(t *testing.T) {
+	t.Skip("No system bus in schroot")
 	oldConn, err := SystemBus()
 	if err != nil {
 		t.Error(err)
@@ -66,6 +67,7 @@ func TestConnectSessionBus(t *testing.T) {
 }
 
 func TestConnectSystemBus(t *testing.T) {
+	t.Skip("No system bus in schroot")
 	conn, err := ConnectSystemBus()
 	if err != nil {
 		t.Fatal(err)
